### ISSUE TITLE
Tab "Buttons" are not accessible

### ISSUE THREAD

**Issue by WesWedding** [WesWedding/sta-enhanced#7]
Screen readers are not aware that the tabs exist or serve any functions.  They should be updated to buttons so that they remain a part of the accessibility tree.
